contracts/cw-pre-propose-base-proposal-single/src/contract.rs

```diff
@@ -0,0 +1,108 @@
+#[cfg(not(feature = "library"))]
+use cosmwasm_std::entry_point;
+use cosmwasm_std::{
+    Binary, CosmosMsg, Deps, DepsMut, Empty, Env, MessageInfo, Response, StdResult,
+};
+use cw2::set_contract_version;
+
+use cw_pre_propose_base::{
+    error::PreProposeError,
+    msg::{ExecuteMsg as ExecuteBase, InstantiateMsg as InstantiateBase, QueryMsg as QueryBase},
+    state::PreProposeContract,
+};
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+pub(crate) const CONTRACT_NAME: &str = "crates.io:cw-pre-propose-base-proposal-single";
```

Comment:
this contract name is a bit of a mouthful; could I suggest `cw-pre-propose-single`?